discovery-provider/src/solana/solana_client_manager_unit_test.py

```diff
@@ -126,3 +132,27 @@ def test_get_signatures_for_address(_):
         solana_client_manager.get_signatures_for_address(
             "JsP7ivVoNhQZXMDmQd5m6VP6mYmycdCFHzK3i2jAmT9"
         )
+
+
+@mock.patch("solana.rpc.api.Client")
+def test_get_sol_tx_info_v0_transaction(_):
```

Comment:
I copied the pattern to make sure we have coverage here. This test doesn't provide much value.